canonical-web-and-design/juju.is
# Implement filters on tutorials page

## Done

Implement filters on tutorials page

## QA

- Go to https://juju-is-211.demos.haus/tutorials
- Select/unselect filters as see everything works as expected
- Do the same on mobile view
- Select some filters and refresh the page. See the filters are apply in accordance with the url


## Issue / Card

Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1988

## Screenshots

[if relevant, include a screenshot]
